### ISSUE TITLE
org.id-lifdeposit: assign proxy admin to community multi-sig

### ISSUE THREAD

**Issue by mfw78** [windingtree/org.id-lif-deposit#3]
# tl;dr

Superuser functions for complete contract upgrade are assigned to an `EOA` (nominally an old Winding Tree Deployment / general use wallet. This represents an immediate high level security risk with funds at risk of loss.

# Overview

The [`LifDeposit`](https://etherscan.io/address/0x7Fa6Ad0866Caa4aBdE8d60B905B3Ae60A3E0f014) implementation contains a superuser:

1. [`ProxyAdmin`](https://etherscan.io/address/0x9068a8323b7d6cbc2831c54569981e3ba7b5862e) - responsible for the token logic of the contract. This contract contains an `owner`, which can be viewed [here](https://etherscan.io/address/0x9068a8323b7d6cbc2831c54569981e3ba7b5862e#readContract). Currently the `owner` is set to [`0xead12c6ee973e03d2d630d81efa57f393cb9117f`](https://etherscan.io/address/0xead12c6ee973e03d2d630d81efa57f393cb9117f) which appears to be an old WT Deployment / general use wallet.

Should the deployer EOA key be compromised, it is possible for the token contract to be upgraded and all funds stolen.

Fortunately, a `ProxyAdmin` has already been deployed that is being used by `Lif2` with the correct `owner` set on that `ProxyAdmin`, therefore, we can simplify the process and change the `ProxyAdmin` on `LifDeposit` to the same as that used by `Lif2` (which has the `owner` set the community multi-sig).

Therefore it is **strongly recommended** to immediate execute the following:

```js
{
  chainId: 1,
  data: 0x7eff275e0000000000000000000000007fa6ad0866caa4abde8d60b905b3ae60a3e0f014000000000000000000000000f66ee7f8406ac8922010e8c23ed62d0be05aec5e,
  from: 0xead12c6ee973e03d2d630d81efa57f393cb9117f,
  to: 0x9068a8323b7d6cbc2831c54569981e3ba7b5862e,
  value: { BigNumber: "0" },
}
```

This would in effect deprecate the `ProxyAdmin` at `0x9068a8323b7d6cbc2831c54569981e3ba7b5862e`, with the `LifDeposit` Proxy now being administered by `ProxyAdmin` at `0xf66ee7f8406ac8922010e8c23ed62d0be05aec5e` (whose owner is the community multi-sig).

**Comment by mfw78** [comment]
```js
{
  chainId: 1,
  data: '0xf2fde38b000000000000000000000000876969b13dcf884c13d4b4f003b69229e6b7966a',
  from: '0xead12c6ee973e03d2d630d81efa57f393cb9117f',
  to: '0x9068a8323b7d6cbc2831c54569981e3ba7b5862e',
  value: { BigNumber: "0" },
}
```

**Comment by mfw78** [comment]
`ProxyAdmin` transferred to `0xf66ee7f8406ac8922010e8c23ed62d0be05aec5e`. Refer to https://etherscan.io/tx/0x8286ff3952db941c845a35a530232d6947724348d056bfea62d7361b52a0a922
